var/spack/repos/builtin/packages/fl/package.py

```diff
@@ -13,12 +21,18 @@ class Fl(Package):
     homepage = "https://www.flxpert.hu/fl/"
     url      = "https://www.flxpert.hu/fl/fl_0.79.47-amd64-linux.tar.gz"
 
-    version('0.79.47', sha256='b8a4a74118c1a024313bf912261fbc016a53f2d15adb1226217f2a10a9f7ca9a')
+    for ver, packages in _versions.items():
+        key = "{0}-{1}".format(platform.system(), platform.machine())
+        pkg = packages.get(key)
+        if pkg:
+            version(ver, sha256=pkg[0], url=pkg[1])
 
     def install(self, spec, prefix):
         if (self.spec.satisfies('platform=linux') and
-           self.spec.target.family == 'x86_64'):
+           (self.spec.target.family == 'x86_64' or
+            self.spec.target.family == 'aarch64')):
             with working_dir('fl_{0}'.format(spec.version)):
                 install_tree('.', prefix)
         else:
-            raise InstallError('fl is built for Linux x86_64 platform only.')
+            raise InstallError('fl is built for Linux x86_64 or \
+                                aarch64 platform.')
```

Comment:
Rely on automatic line continuation and string  concatenation, or just reword slightly to fit it :) 
```suggestion
            raise InstallError('fl requires Linux x86_64 or aarch64 platforms.')
```